packages/server/database/types/Organization.ts

```diff
@@ -14,6 +14,9 @@ interface Input {
   updatedAt?: Date
   showConversionModal?: boolean
   payLaterClickCount?: number
+  tierLimitExceededAt?: Date
+  scheduledLockAt?: Date
+  lockedAt?: Date
```

Comment:
+1 I would imagine we never set these on construction and thus could omit them from the Input object

Reply:
Make sense

Reply:
You just removed the assignment, now it's broken because you could add it to the `Input` object but it would not get assigned anymore in the constructor